js/everything.js

```diff
@@ -448,12 +448,19 @@ $(function() {
     });
 });
 
-/* jshint ignore:start */
+/*eslint-disable */
```

Comment:
Disables eslint just for `\\Google Analytics goodness` part. Otherwise numerous rules would be broken and we'll have to expand that block much more.